### DISCUSSION TITLE
Styling disabled entries

### DISCUSSION THREAD

**Discussion by Mads-Ottendal** [azeezat/react-native-select#68] [Q&A] [answered]
Hello, 
Is it in any way possible to style **only** the disabled objects in a list.
``` 
options={[
              {
                title: 'rooms
                data: [
                  { label: 'Room 1', value: 'A' },
                  { label: 'Room 2', value: 'B' },
                  { label: 'Room 3', value: 'C' },
                  { label: 'Room 4', value: 'D', disabled: true},
                ],
              },
            ]}
```
So above i want to style the css for only Room 4 in the checkbox. For example when a room is disabled its checkbox is red instead of green like the others.

Any help on how to do this if it is possible would be greatly appreciated :)

**Comment by azeezat** [comment]
@Mads-Ottendal At this moment, the feature  isn't available. However, we really appreciate your feedback and interest in improving this package.

**Reply by azeezat** [reply, marked as answer]
Thanks for reporting this issue. A new version has been published. Please add the following props

```js
  checkboxControls={{
              checkboxDisabledStyle: {
                borderColor: 'red',
                backgroundColor: 'red',
              },
            }}
```
